src/app/http/http.service.ts

```diff
@@ -1,7 +1,28 @@
+import { Movie } from './../interfaces';
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import * as YoutubeResponseInterfaces from './../YoutubeResponseInterface';
```

Comment:
importujez tu wszystkie interfejsy jako obiekt nazwany YoutubeResponseInterfaces, to nie ma sensu. Zmień Welcome (skąd ta nazwa w ogóle? pewnie jakiś generator domyślnie Ci wypluł...) na ytResponse i zaimportuj tylko ten jeden interfejs